commands/operator-sdk/cmd/scorecard/scorecard.go

```diff
@@ -190,34 +178,21 @@ func ScorecardTests(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to decode custom resource manifest into object: %s", err)
 	}
+	if err := waitUntilReady(obj); err != nil {
+		return fmt.Errorf("failed waiting for CR to be ready: %v", err)
+	}
+	var suites []*TestSuite
 	if viper.GetBool(BasicTestsOpt) {
-		fmt.Println("Checking for existence of spec and status blocks in CR")
-		err = checkSpecAndStat(runtimeClient, obj, false)
-		if err != nil {
-			return err
-		}
-		// This test is far too inconsistent and unreliable to be meaningful,
-		// so it has been disabled
-		/*
-			fmt.Println("Checking that operator actions are reflected in status")
-			err = checkStatusUpdate(runtimeClient, obj)
-			if err != nil {
-				return err
-			}
-		*/
-		fmt.Println("Checking that writing into CRs has an effect")
-		logs, err := writingIntoCRsHasEffect(obj)
-		if err != nil {
-			return err
-		}
-		log.Debugf("Scorecard Proxy Logs: %v\n", logs)
-	} else {
-		// checkSpecAndStat is used to make sure the operator is ready in this case
-		// the boolean argument set at the end tells the function not to add the result to scTests
-		err = checkSpecAndStat(runtimeClient, obj, true)
-		if err != nil {
-			return err
+		conf := BasicTestConfig{
+			Client:        runtimeClient,
+			CR:            obj,
+			ProxyPod:      proxyPod,
+			Timeout:       60,
+			RetryInterval: 5,
```

Comment:
I don't see where `Timeout` and `RetryInterval` are used by the basic tests? Are we planning to use those in the future?

Reply:
A couple of the basic tests originally needed that, but I removed one (actions reflected in status) and changed the other so it doesn't require that anymore (checkSpecAndStat). I'll remove these 2 variables